apps/documentation/src/app/app.module.ts

```diff
@@ -19,25 +28,23 @@ import {
 import { MockDataLayerService } from './mock/mock-data-layer.service';
 import { MockPatientService } from './mock/mock-patient.service';
 import { NgDocIconComponent, NgDocButtonIconComponent } from '@ng-doc/ui-kit';
+import { HttpClientModule } from '@angular/common/http';
 
 @NgModule({
   declarations: [AppComponent],
   imports: [
     BrowserModule,
-    AppRoutingModule,
     BrowserAnimationsModule,
-    NgDocNavbarModule,
-    NgDocSidebarModule,
+    NgDocRootComponent,
+    NgDocNavbarComponent,
+    NgDocSidebarComponent,
     NgDocIconComponent,
     NgDocButtonIconComponent,
+    FhirChartLegendModule,
```

Comment:
should `FhirChartLegendModule` be imported in the standalone components instead?